packages/desktop/electron/lib/menu.js

```diff
@@ -80,7 +80,7 @@ const buildTemplate = () => {
                 {
                     label: `${state.strings.checkForUpdates}...`,
                     click: () => getOrInitWindow('main').webContents.send('menu-check-for-update'),
-                    enabled: app.isPackaged ? state.enabled : false,
+                    enabled: app.isPackaged && features?.electron?.autoUpdates?.enabled ? state.enabled : false,
```

Comment:
This popup is essentially a version popup so we still want to enable getting to it